### ISSUE TITLE
Midterm Peer Review - vvs24

### ISSUE THREAD

**Issue by virenvshah** [mianyangSharon/Intersection-Congestion-Prediction#9]
The project I am reviewing is about predicting the traffic congestion at major intersections in US cities.     The dataset consisted of commuting metrics from vehicles in 4 different cities.  The dataset seemed big and messy and the project was very interesting.  

The things I liked:

1.  I liked that you considered corner cases when analyzing their dataset.  Although your dataset had a "cities" feature, you considered the possibility that the roads could link to one another and you decided to use K-means clustering on the latitude and longitude to verify that the roads are in fact far away from one another.  

2.  I really liked your graphs which showed how the number of busy intersections increased and then decreased depending on the hour of the day.  This visual really helped me understand your dataset.

3.  I think your upcoming plan of clustering intersections within the same city is very interesting.   It would be cool to find other unique relationships between these intersections that you may not have considered before.  Maybe try clustering the intersections based on hour and total stopped time.  This would help you understand which intersections are busy at the same time.  

Things to consider:

1.  When running your models (for example linear regression) do not just report a mean squared error, because it doesn't have much meaning since the error value is specific to the features and y label used in your project.  What would be more meaningful would be to split your data into a training set and test set (describe how large each set is), train your linear regression model on the training set, and then report accuracy, precision and recall statistics on your test set.  

2.  It might be easier to also train some binary classification models on your dataset.  You defined busy as any street who's 50th percentile of total time stopped is greater than 10.  Label all these streets with 1 and all other streets as -1 and run a few binary classification models.  If you can get decent accuracy with linear regression, your binary classifiers will be really good.

3.  In your introduction, you describe that you want to predict congestion at intersections.  I would recommend describing your definitions of busyness in the introduction and possibly describe other methods that define congestion (for example multiple busy intersections that are located in the same area could define congestion in that area).
